include/operations/blas_constants.h

```diff
@@ -147,27 +147,52 @@ struct constant<value_t, const_val::abs_min> {
 };
 
 template <typename value_t, typename index_t>
-struct constant<IndexValueTuple<value_t, index_t>, const_val::abs_max> {
-  constexpr static SYCL_BLAS_INLINE IndexValueTuple<value_t, index_t> value() {
-    return IndexValueTuple<value_t, index_t>(
+struct constant<IndexValueTuple<index_t, value_t>, const_val::abs_max> {
+  constexpr static SYCL_BLAS_INLINE IndexValueTuple<index_t, value_t> value() {
+    return IndexValueTuple<index_t, value_t>(
         std::numeric_limits<index_t>::max(),
         std::numeric_limits<index_t>::max());
   }
 };
 
 template <typename value_t, typename index_t>
-struct constant<IndexValueTuple<value_t, index_t>, const_val::abs_min> {
-  constexpr static SYCL_BLAS_INLINE IndexValueTuple<value_t, index_t> value() {
-    return IndexValueTuple<value_t, index_t>(
+struct constant<IndexValueTuple<index_t, value_t>, const_val::abs_min> {
+  constexpr static SYCL_BLAS_INLINE IndexValueTuple<index_t, value_t> value() {
+    return IndexValueTuple<index_t, value_t>(
+        std::numeric_limits<index_t>::max(), 0);
+  }
+};
+
+template <typename value_t, typename index_t>
+struct constant<IndexValueTuple<index_t, value_t>, const_val::max> {
+  constexpr static SYCL_BLAS_INLINE IndexValueTuple<index_t, value_t> value() {
+    return IndexValueTuple<index_t, value_t>(
+        std::numeric_limits<index_t>::max(),
+        std::numeric_limits<index_t>::max());
```

Comment:
Shouldn't the second one be `std::numeric_limits<value_t>::max()`?

Reply:
Correct, changed now.